### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by toxicFork**
Fixed a few typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ C++17 is needed to compile the project.
 
 The scene description and assets are taken from [GLSL-PathTracer](https://github.com/knightcrawler25/GLSL-PathTracer) project [4]. The whole dataset can be downloaded from [link](https://drive.google.com/file/d/1UFMMoVb5uB7WIvCeHOfQ2dCQSxNMXluB/view).
 
-Download the assets folder and place it in `PBRVulkan/Assets/Scenes/`. The folder strucutre has to be as follows:
+Download the assets folder and place it in `PBRVulkan/Assets/Scenes/`. The folder structre has to be as follows:
```
Comment (Zielon) [inline README.md:35]: `typo`

**Review by Zielon — CHANGES_REQUESTED** [review]
Hey, I found one `typo`.

File: README.md (hunk 1 above)
Comment (toxicFork) [reply README.md:35]: Haha, typo tax

**Review by Zielon — APPROVED** [review]
Thank you.

**Pull request merged**